app/generic/verifier/facades/acapy_verifier_v1.py

```diff
@@ -45,6 +45,51 @@ async def create_proof_request(
         )
         return utils.record_to_model(presentation_exchange)
 
+    @classmethod
+    async def get_creds(cls, controller: AcaPyClient, pres_ex_id: str):
+        try:
+            proof_id = utils.pres_id_no_version(proof_id=pres_ex_id)
+            return await controller.present_proof_v1_0.get_matching_credentials(
+                pres_ex_id=proof_id
+            )
+        except Exception as e:
+            logger.error(f"{e!r}")
+            raise e from e
+
+    @classmethod
+    async def get_proofs(
+        cls, controller: AcaPyClient, pres_ex_id: Optional[str] = None
```

Comment:
I would split this method up in two separate methods. One for get proofs and one for get proof by id. This is also how we did it for issuer, and also how the ACA-Py API works.

Reply:
Also I think we should be consistent in using `pres_ex_id` vs `proof_id`

Reply:
sure, fine by me. Put that into one because I thought that's more convenient, but have no preference here/happy to split it into 2.

Reply:
> Also I think we should be consistent in using `pres_ex_id` vs `proof_id`

This btw I chose because the PresentationExchange uses proof_id. Thought this would make more sense to the user when getting a PresentationExchange with a proof ID (that contains v1- or v2-) to put that in knowing what is meant. Under the hood it's still pres_ex_id for the background methods. Not sure what's best here. Any preference on how to go about this @TimoGlastra?